tests/system/feature_store/test_spark_engine.py

```diff
@@ -1513,8 +1513,9 @@ def test_as_of_join_result(self):
             spark_service=self.spark_service,
             target=target,
         )
-        spark_engine_res = resp.to_dataframe()
+        spark_engine_res = resp.to_dataframe().sort_index(axis=1)
 
-        assert local_engine_res.sort_index(axis=1).equals(
-            spark_engine_res.sort_index(axis=1)
-        )
+        self._logger.info(f"result of LOCAL engine merger: \n {local_engine_res}")
+        self._logger.info(f"result of SPARK engine merger: \n {spark_engine_res}")
```

Comment:
A bit pedantic but...
```suggestion
        self._logger.info(f"result of LOCAL engine merger:\n  {local_engine_res}")
        self._logger.info(f"result of SPARK engine merger:\n  {spark_engine_res}")
```